tests/pxcentral/pxcentral_test.go

```diff
@@ -0,0 +1,68 @@
+package tests
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultTimeout       = 5 * time.Minute
+	defaultRetryInterval = 20 * time.Second
+	appReadinessTimeout  = 20 * time.Minute
```

Comment:
Does px-central takes upto 20 mins to become ready? If not can we reduce this timeout?

Reply:
It takes nearly 10 mins.